styles/styles.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  width: 310px;
+  height: 50px;
+  margin-top: 120px;
+  margin-right: auto;
+  margin-left: auto;
```

Comment:
You use `text-align: center` and it works. So, are there reasons to write `margin-left/right: auto`?